tuneme/forms.py

```diff
@@ -0,0 +1,38 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+
+class LoginForm(forms.Form):
+    username = forms.CharField(widget=forms.TextInput(attrs={
+        'placeholder': _('Username'),
+    }))
+    password = forms.CharField(widget=forms.PasswordInput(attrs={
+        'placeholder': _('4 Digit PIN'),
+    }))
+
+
+class RegistrationForm(forms.Form):
+    username = forms.CharField(widget=forms.TextInput(attrs={
+        'placeholder': _('Username'),
+    }))
+    password = forms.CharField(widget=forms.PasswordInput(attrs={
+        'placeholder': _('4 Digit PIN'),
+    }))
```

Comment:
I think the registration form needs a date of birth? I'm not entirely sure, @bruskiza please check this in the SoW and confirm?